libs/elements/src/lib/form/vl-form.element.ts

```diff
@@ -44,6 +53,35 @@ export class VlFormElement extends BaseElementOfType(HTMLFormElement) {
         this._disableNativeValidation();
     }
 
+    /**
+     * this function creates a MutationObserver that will observe the elements added to the form
+     * if it detects that a form validation element has been added, it'll undress & dress the form again
+     */
+    _observeAddedElements(): void {
+        const node = this as unknown as Node;
+        const vlFormElement = node as unknown as VlFormElement;
+        const observer = new MutationObserver((mutations) => {
+            for (const mutation of mutations) {
+                const { addedNodes } = mutation;
```

Comment:
Check je nog even af of de event listeners van de removedNodes automatisch verwijderd worden?